### DISCUSSION TITLE
Allow server side use / Expose api_key

### DISCUSSION THREAD

**Discussion by ggcaponetto** [Glench/ExtPay#123] [General]
### What I want to do
I would like to have a backend service that performs task if a user has a paid subscription.

Currently the `extpay` package does not run outside of the browser environment. I thought of writing my own node.js script that fetches the user in my own backend like this library does:

```js
const resp = await fetch(`${EXTENSION_URL}/api/user?api_key=${api_key}`, {
            method: 'GET',
            headers: {
                'Accept': 'application/json',
            }
        })
```

For this reason I would like the `api_key` to be exposed by a new the `gApiKey()` function.

I would be happy to work on a PR if this makes sense to the maintainer and/or ther users.

### Workaround
Get the key yourself.
```js
chrome.storage.sync.get(['extensionpay_api_key']).then((res) => {
    console.log(res)
})
```

Cheers

Giuseppe

**Comment by Glench** [comment]
Hm, I'm not sure I want to expose endpoints like this as a feature, but it's fine if folks want to do it. If others have a need for this please share your use case. 

And @ggcaponetto if you already have a server why not just make your own payments system?

**Reply by ggcaponetto** [reply]
This library saves me a lot of work even If I have my own server! My current server implementation only checks if the user has paid. It is a couple of lines only using the workflow described above. I am very happy so far!
